examples/siro_sandbox/configs/social_nav_study.yaml

```diff
@@ -0,0 +1,291 @@
+# @package _global_
+
+defaults:
+  - /benchmark/rearrange: multi_agent_tidy_house_fp_spot_humanoid
```

Comment:
I suspect you are copy-pasting a ton of configs between this and fetch.yaml. I still suggest a base sandbox config yaml for stuff that is fundamentally required for the sandbox app (and not specific to an app state). This is conceptually the same as how we have lots of shared code in sandbox_driver.py that is shared between app states (because that code is fundamental to the sandbox app, regardless of app state).